src/main/scala/viper/gobra/reporting/Message.scala

```diff
@@ -29,6 +30,11 @@ sealed trait GobraVerificationResultMessage extends GobraMessage {
   val verifier: String
 }
 
+case class VerificationTaskFinishedMessage(taskName: String) extends GobraMessage {
```

Comment:
What is the difference between `VerificationTaskFinishedMessage` and `GobraOverallSuccessMessage` resp. `GobraOverallFailureMessage`?

Reply:
I see, thanks! Please add a comment to this case class to explain this difference